custom_components/elasticsearch/es_doc_creator.py

```diff
@@ -29,6 +29,17 @@
 from custom_components.elasticsearch.system_info import SystemInfo
 
 ALLOWED_ATTRIBUTE_TYPES = tuple | dict | set | list | int | float | bool | str | None
+SKIP_ATTRIBUTES = [
+    "friendly_name",
+    "id",
+    "type",
+    "entity_picture",
+    "icon",
+    "entity_id",
+    "device_class",
+    "state_class",
+    "unit_of_measurement",
```

Comment:
Where do these fields exist in other forms? I don't see them represented elsewhere in my documents.

Reply:
They are added in the floors and labels PR, this PR would merge after floors and labels